agent/util-scripts/testit

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "\$*='${*}'"
+echo "\$@='${@}'"
```

Comment:
perl has nothing over bash in unreadability...

Reply:
oops ... fixed.